src/crypto.ts

```diff
@@ -10,7 +10,9 @@ export function hash(data: string, salt: string): string {
   return hashedString;
 }
 
-export function signJWT(userId: number) {
-  const token = jwt.sign({ userID: userId }, privateKey, { expiresIn: 200 });
+export function signJWT(userId: number, rememberMe: boolean) {
+  let expirationSeconds: number;
+  rememberMe ? (expirationSeconds = 604800) : (expirationSeconds = 120);
```

Comment:
Aqui poderia usar const direto inline:

```suggestion
  const expirationSeconds = rememberMe ? 604800 : 120;
```

Melhor né? Sobre a tipagem, esses tipos imitivos o Typescript infere e já passa a considerar que é um `number` pelo valor passado no ternário.